src/plugins/discover/public/mocks.tsx

```diff
@@ -32,4 +33,5 @@ const createStartContract = (): Start => {
 export const discoverPluginMock = {
   createSetupContract,
   createStartContract,
+  getMockDiscoverStateContainer,
```

Comment:
Would we be able to reuse/adapt `src/plugins/discover/public/__mocks__/discover_state.mock.ts` for this instead of having a second version?

Reply:
You are right.

I have deleted my implementation and now I am using this one.